app/views/posts/new.html.erb

```diff
@@ -17,6 +17,7 @@
 
     <%= form.hidden_field :latitude %>
     <%= form.hidden_field :longitude %>
+    <%= form.file_field :image %><br>
```

Comment:
very nitpicky thing but group the non-hidden fields together if you can (put them above the hidden_fields here)